c/driver/framework/base_driver.cc

```diff
@@ -66,7 +66,24 @@ Result<std::string_view> Option::AsString() const {
         if constexpr (std::is_same_v<T, std::string>) {
           return value;
         }
-        return status::InvalidArgument("Invalid string value {}", *this);
+        return status::InvalidArgument("Invalid string value {}", this->Format());
+      },
+      value_);
+}
+
+std::string Option::Format() const {
+  return std::visit(
+      [&](auto&& value) {
+        using T = std::decay_t<decltype(value)>;
+        if constexpr (std::is_same_v<T, adbc::driver::Option::Unset>) {
+          return std::string("(NULL)");
+        } else if constexpr (std::is_same_v<T, std::string>) {
+          return std::string("'") + value + "'";
```

Comment:
```suggestion
          return (std::stringstream() << std::quoted(value)).str();
```

Reply:
I thought the general advice is that stringstream is really slow? (It's also another header...) I'd rather keep the concat